asynq/profiler.py

```diff
@@ -23,3 +24,9 @@ def append(stats):
 
 def reset():
     _state.stats[:] = []
+    _state.counter = 0
+
+
+def incr_counter():
+    _state.counter = _state.counter + 1
```

Comment:
This has a race condition: we could switch threads between the time we read the attribute and the time we write to it. Is it important for the ids to be fully unique?